.github/workflows/build.yml

```diff
@@ -1,6 +1,10 @@
 name: Build master
 
-on: [push, pull_request, workflow_dispatch]
```

Comment:
workflow_dispatch is used [for passing inputs to the workflow](https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#onworkflow_dispatch). I don't think that we need it.